wallet/src/actors/storage/mod.rs

```diff
@@ -0,0 +1,69 @@
+//! # Storage actor
+//!
+//! It is charge of managing the connection to the key-value database. This actor is blocking so it
+//! must be used with a `SyncArbiter`.
+
+use std::path::PathBuf;
+
+use actix::prelude::*;
+
+mod error;
+mod handlers;
+
+pub use error::Error;
+pub use handlers::*;
+
+/// Storage actor.
+pub struct Storage {
+    /// Path where to store the wallet database.
+    /// This is used during the wallet genration step.
+    // TODO: remove this allow attribute.
```

Comment:
Does this still hold?

Reply:
Yes, I will improve the description though